src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,212 @@
+package duke.command;
+
+import duke.common.Messages;
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.Task;
+import duke.task.ToDos;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Parser {
+    public Parser() {
+    } //constructor
+
+    protected static int orderAdded = 1;
+    protected boolean notBye = true;
```

Comment:
You can consider changing the boolean name to sound more like a boolean. notBye might cause some confusion because of the negative reference, perhaps something more obvious to the developer at first glance can improve readability?